### ISSUE TITLE
IsLabelledNewick="true" or "false" ?

### ISSUE THREAD

**Issue by walterxie** [LinguaPhylo/LPhyBeast#172]
In TimeTreeToBEAST, should it uses IsLabelledNewick="true" or "false" as default:

BEAST 2 doc https://www.beast2.org/2014/07/28/all-about-starting-trees.html uses `IsLabelledNewick="true"`, but this will cause the trouble when caring about the internal node index. Use "false", this problem will be solved.

**Comment by alexeid** [comment]
Surely false is wrong. Because in general we will not use only numbers for taxa names in lphy.

**Comment by walterxie** [comment]
https://github.com/CompEvol/beast2/issues/1173

**Comment by walterxie** [comment]
Yes, I think we should leave this unchanged and see how the above beast development will improve the internal node id  issue ...
